doc/tutorials/enable_ivshmem.rst

```diff
@@ -153,7 +152,6 @@ Linux-based post-launched VMs (VM1 and VM2).
         :emphasize-lines: 5
```

Comment:
same here:
```suggestion
        :emphasize-lines: 4
```

Reply:
> I _think_ there are some instances of these left in:
> 
> * doc/developer-guides/hld/hostbridge-virt-hld.rst
> * doc/tutorials/gpu-passthru.rst

Done.

> * doc/tutorials/launch_uos.sh

I will check whether we need remove instances from *.sh of doc dirs.
Then update this patch.

> * misc/config_tools/data/sample_launch_scripts/apl-up2/launch_uos.sh
> * misc/config_tools/data/sample_launch_scripts/nuc/launch_uos.sh
> * misc/config_tools/data/sample_launch_scripts/apl-mrb/launch_uos.sh

These will be a split patch for config tool.